modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+
+* With this update, the following new features have been introduced for multi-sourced application:
+** Users can access revision history and rollback to a specific version of a multi-source application. Specifically, the *Details* view's panel displays the most recent revision or commit details for each source in the multi-source application. Also, the *History* and *Rollback* panel displays the revision history associated with each source, enabling users to select the desired version for rollback. link:https://issues.redhat.com/browse/GITOPS-4647[GITOPS-4647]
+** A new *SOURCES* tab is introduced for viewing and editing the source parameters of a multi-source application. The *SOURCES* tab displays all source parameters associated with the multi-source application, allowing you to edit the parameter values by using a form in the Web UI. This functionality consists of the capabilities for single-source applications, which are now extended to accommodate multiple sources. link:https://issues.redhat.com/browse/GITOPS-4192[GITOPS-4192]
+
+* With this update, Repo server storage used for storing manifests can be extended by mounting Persistent Volumes. The default storage volume utilized by the Repo server is `EmptyDir`, which has limitations and may prove inadequate in scenarios involving numerous repositories, sizable repositories, or repositories containing large manifests. Additional storage can be mounted when required using the `.spec.repo.volumes` and `.spec.repo.volumeMounts` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-4646[GITOPS-4646]
++
+For more information, refer to the link:https://argocd-operator.readthedocs.io/en/latest/usage/ha/repo-server/#high-availability-for-repo-server[HA for Repo Server].
+
+* With this update, users can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
+
+* With this update, users can expose Argo CD Application labels on Kubernetes events generated by Argo CD. When events are created for applications that have specific label keys defined in the `resource.includeEventLabelKeys` field within the `argocd-cm` ConfigMap, the controller associates the relevant labels with these events. This enhancement facilitates the filtering and processing of events according to application labels. To set the `resource.includeEventLabelKeys` field in a {gitops-shortname} environment, use the `.spec.extraConfig` field in the Argo CD CR. The {gitops-shortname} Operator automatically incorporates these fields from the `.spec.extraConfig` field into the `argocd-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-3233[GITOPS-3233]
```

Comment:
If you move the GitOps link to after the content in the entry above, I'd move it here too.

Reply:
I would not move the JIRA entry as I would like to stick to how we are linking currently in GitOps.